replication.md

```diff
@@ -18,25 +18,25 @@ Dqlite configures SQLite to use a custom VFS implementation which stores files i
 
-Earlier versions of Dqlite had a simpler custom VFS than the current one. It was used purely as in-memory storage to avoid persisting data twice (in a SQLite file and in the Raft log) , and had no extra functionality for "watching" SQLite's I/O and extracting transactions. Instead, Dqlite relied on a patch to be applied to the upstream SQLite source code, introducing various hooks during a transaction lifecycle and allowing Dqlite to extract the information it needed for replication. That approach has been abandoned because SQLite is [Open-Source, not Open-Contribution](https://www.sqlite.org/copyright.html) and it is very difficult to get any changes merged upstream, for good and for bad.
+Earlier versions of Dqlite had a simpler custom VFS than the current one. It was used purely as in-memory storage to avoid persisting data twice (in a SQLite file and in the Raft log) , and had no extra functionality for "watching" SQLite's I/O and extracting transactions. Instead, Dqlite relied on a patch to be applied to the upstream SQLite source code, introducing various hooks during a transaction life cycle and allowing Dqlite to extract the information it needed for replication. That approach has been abandoned because SQLite is [Open-Source, not Open-Contribution](https://www.sqlite.org/copyright.html) and it is very difficult to get any changes merged upstream, for good and for bad.
 
 ## Write-Ahead Log
 
 Dqlite also configures SQLite to use a [Write-Ahead Log](https://sqlite.org/wal.html) (WAL) for implementing atomic commits and rollbacks.
 
-In this mode SQLite uses two files to store data: one is the **main database file**, and the other is the **WAL file**. When a SQLite transaction that modifies the databaste wants to commit those changes, it does not write the database pages it has modified directly to the database file, but it rather appends them to the WAL file. The last page of a write transaction appended to the WAL will contain a **commit marker** that marks the end of the transaction. Once the WAL file grows too big, it needs to be "checkpointed" and the latest version of each page of each committed transaction it contains gets copied back to the main database file.
+In this mode SQLite uses two files to store data: one is the *main database file*, and the other is the *WAL file*. When an SQLite transaction that modifies the database wants to commit those changes, it does not write the database pages it has modified directly to the database file, but it rather appends them to the WAL file. The last page of a write transaction appended to the WAL will contain a *commit marker* that marks the end of the transaction. Once the WAL file grows too big, a checkpoint must be created, and the latest version of each page of each committed transaction it contains gets copied back to the main database file.
```

Comment:
suggest to change "a checkpoint must be created," to "a checkpoint operation must be performed,"